modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -411,6 +412,7 @@ import com.squareup.moshi.adapter
         val accept = response.header(ContentType)?.substringBefore(";")?.lowercase(Locale.US)
 
         // TODO: handle specific mapping types. e.g. Map<int, Class<?>>
+        @Suppress("UNNECESSARY_SAFE_CALL")
```

Comment:
The `response.body` is non-nullable in recent okhttp3, but not with older ones. Added this to keep the `?.` but silence the linters.